markdown/what.md

```diff
@@ -1,23 +1,79 @@
+We opted to deploy the platform on cloud server instances using Ansible playbooks.
+
+To achieve a cloud-centric and image-driven deployment, we incorporated OpenStack Heat into our workflow and managed our clusters as Heat stacks. Additionally, we effectively managed the entire system using GitLab CI, leveraging its capabilities to orchestrate the deployment process.
+
 
 ## Tutor
 
+<!-- Note -->
+
+Tutor is a community-supported Docker-based distribution that simplifies the deployment, customization, debugging, upgrading, and scaling of Open edX platforms, starting from the Lilac release. It completely replaced the Open edX native method of using Ansible playbooks for Open edX installation since the Maple release.
+
+So why Tutor came into the picture?
+
+As per the community, it was difficult to install Open edX with native installation scripts. For instance, there are no official instructions for upgrading an existing platform to the new release. There is a recommended approach to backup the data, trash the existing server and create a new one. As a consequence, people keep running deprecated releases and are hesitant to upgrade their platforms.
+
+Tutor aims to simplify the installation and upgrade process of Open edX platforms. Here's how Tutor simplifies Open edX deployment:
+
+* Application Isolation: Tutor runs Open edX processes inside Docker containers, providing isolation and encapsulation of the application components.
+
+* Command Line Interface (CLI): Tutor offers a CLI for common administrative tasks, making it easier to manage and control the Open edX platform.
+
+* Plugin System: Tutor utilizes a plugin-driven system that allows users to extend or customize Open edX without modifying the core codebase.
+
+Tutor provides comprehensive documentation that offers detailed information on how to use the distribution effectively. If you want to explore further, you can refer to the official Tutor documentation for more insights.
 
 ## Our goals
+
+<!-- Note -->
+
+In order to adopt the new Tutor deployment method, our aim was to identify a solution that aligned with the following key goals:
+
+* CI driven Open edX deployment: Our objective was to achieve a fully automated deployment of the Open edX platform to a Magnum managed Kubernetes cluster, similar to our previous setup, through a CI/CD pipeline.
+
+* Configuration and setup changes via Gerrit/Zuul: All configuration and setup changes to Open edX environment are to be done through our Gerrit/Zuul toolchain.
+
+* High Availability Clusters across Two Regions with Data Replication: We wanted to have a deployment model that utilized high availability clusters distributed over two different regions, while ensuring data replication between them, as we had done in our previous setup. This would provide redundancy and fault tolerance for uninterrupted service.
```

Comment:
Data replication and redundancy: yes, high availability: not really, fault tolerance: no. Yes, Kubernetes gives us HA _within_ a region, but our HA cluster doesn't span two regions.